java-client/build.gradle.kts

```
@@ -171,8 +171,8 @@ publishing {
 dependencies {
     // Compile and test with the last 7.x version to make sure transition scenarios where
     // the Java API client coexists with a 7.x HLRC work fine
-    val elasticsearchVersion = "7.17.1"
-    val jacksonVersion = "2.13.1"
+    val elasticsearchVersion = "7.17.3"
+    val jacksonVersion = "2.13.2"
```

A new version of Jackson has been released:

```suggestion
    val jacksonVersion = "2.13.3"
```